NineChronicles.Headless/GraphTypes/TransactionHeadlessQuery.cs

```diff
@@ -101,6 +104,15 @@ public TransactionHeadlessQuery(StandaloneContext standaloneContext)
                                 return false;
                             }
 
+                            if (txStatusFilter is not null)
+                            {
+                                var txResult = TxResult(standaloneContext, tx.Id) as TxResult;
+                                if (txResult is not null && !txStatusFilter.Contains(txResult.TxStatus))
```

Comment:
We should think about the situation where there is no `TxResult`. If `txStatusFilter` is given but there is no `txResult` for txs, shouldn't we ignore the transaction? Or not?

Reply:
@.@ We can ignore this issue since txStatusFilter is nullable, which was designed to handle cases where there is no txResult.